dynawo/sources/Models/Modelica/Dynawo/Electrical/Events/NodeFault.mo

```diff
@@ -21,7 +21,8 @@ model NodeFault "Node fault which lasts from tBegin to tEnd"
   import Dynawo.NonElectrical.Logs.Timeline;
   import Dynawo.NonElectrical.Logs.TimelineKeys;
 
-  Connectors.ACPower terminal;
+  Dynawo.Connectors.ACPower terminal annotation(
```

Comment:
Only Connectors. instead of Dynawo.Connectors. ?

Reply:
done... it is automatically overwritten by OM...